### DISCUSSION TITLE
Student Developer pack

### DISCUSSION THREAD

**Discussion by bhargav2550** [community/community#39215] [GitHub Education]
What to do after verification ,can i access offers now?

**Comment by DSDmark** [comment]
Hi @bhargav2550,

After your verification you can access several offers which is **GitHub Developer** pack provide like.

- Access to **Microsoft** Azure cloud services and learning resources – no credit card required.
- **DigitalOcean** Simple cloud hosting. Enjoy $200 in platform credit for 1 year!
- **Namecheap** 1 year domain name registration on the .me TLD.
- **Heroku** Enjoy a credit of $13 USD per month for 12 months.
- And so on...

Check out this:- [**GitHub Pack**](https://education.github.com/pack)
